trollmoves/client.py

```diff
@@ -615,7 +624,8 @@ def _request_files(huid, destination, login, publisher, **kwargs):
 
         req, fake_req = create_push_req_message(msg, _destination, login)
         LOGGER.info("Requesting: %s", str(fake_req))
-        local_dir = create_local_dir(_destination, kwargs.get('ftp_root', '/'))
+        local_dir = create_local_dir(_destination, kwargs.get('ftp_root', '/'),
+                                     create_target_directory=kwargs.get('create_target_directory', True))
```

Comment:
I'd rather have the test here, otherwise it will be a bit strange, right?

```python
...
result = do_something(args, do_it=False)
...
```

Reply:
Done.